src/project-2.js

```diff
@@ -3,6 +3,8 @@
 const getBiggest = (x, y) => {
   // x and y are integers.  Return the larger integer
   // if they are the same return either one
+  if (x > y) return x;
+  return y;
 };
 
 const greeting = (language) => {
```

Comment:
You can drop English if your base case and English are the same. You could also consider making use of switch cases.